src/game/savegame_bson.c

```diff
@@ -969,14 +1008,18 @@ bool Savegame_BSON_LoadFromFile(MYFILE *fp, GAME_INFO *game_info)
         goto cleanup;
     }
 
-    if (!Savegame_BSON_LoadStartInfo(
-            json_object_get_array(root_obj, "start_info"), game_info)) {
+    if (!Savegame_BSON_LoadResumeInfo(
+            json_object_get_array(root_obj, "start_info"), game_info->start)) {
         goto cleanup;
     }
 
-    if (!Savegame_BSON_LoadEndInfo(
-            json_object_get_array(root_obj, "end_info"), game_info)) {
-        goto cleanup;
+    if (!Savegame_BSON_LoadResumeInfo(
+            json_object_get_array(root_obj, "current_info"),
+            game_info->current)) {
+        if (!Savegame_BSON_LoadCurrentInfo(
```

Comment:
We need to also copy `start` to `current` because with this approach only stats get filled with meaningful values and Lara probably will have her items only from the start of the level for legacy saves.